ext/opentelemetry-ext-psycopg2/tests/test_psycopg2_integration.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import mock
+
+import psycopg2
+
+from opentelemetry import trace as trace_api
+from opentelemetry.ext.psycopg2 import trace_integration
+
+
+class TestPostgresqlIntegration(unittest.TestCase):
+    def test_trace_integration(self):
+        tracer = trace_api.Tracer()
+        with mock.patch("psycopg2.connect"):
+            trace_integration(tracer)
+            cnx = psycopg2.connect(database="test")
+            self.assertIsNotNone(cnx.cursor_factory)
```

Comment:
I believe the test for `trace_integration` should include to see if the functions are successfully decorated. 
As well, the query functions themselves and the `Cursor` should also have tests.
See [OpenCensus](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-postgresql/tests/test_postgresql_trace.py#L23)

Reply:
I'm only decorating the connect method and this will add the cursor factory, not sure if wrapt will add some "decorated" property, psycopg2 internally will instantiate PsycopgTraceCursor that just call dbapi traced_execution that is already tested, unit tests are hard in here because if the mixed Python/c execution, I believe functional tests would be way more useful and easier to implement, did't want to add them yet until #340 is complete and people are ok with that kind of approach